src/QsCompiler/TextProcessor/QsKeywords.fs

```diff
@@ -304,6 +304,9 @@ let qsNEGop = QsOperator.New("-", 45, false)
 let qsSetUnion = QsOperator.New("+", 10, true)
 let qsSetIntersection = QsOperator.New("*", 20, true)
 
+let qsLambdaOp = QsOperator.New("=>", 0, false)
```

Comment:
Seeing that this has not yet been in it so far, I need to double check the parser, and whether it hardcodes the arrows. The precedence here needs to be examined carefully. Could you please add to the task list for the lambda issue to clarify the (pseudo-)precedence of the lambda operator, and remove it from here until we do that (and hard code it in the code write for now) - I can take a look as part of the PR around the data structure that I have in work.

Reply:
Okay, I've commented the QsKeywords.fs ones out and added a ToDo. I've added local hardcoded the ones in code writer with another ToDo.